Description: Remove unavailable features
 As we use Debian packaged libzstd not the bundled, some features are
 not (can not be) enabled at build time and unavailable, or made no-op:
 .
 - debug, fat-lto, no_asm, thin, thin-lto: removed in zstd-sys
 - bindgen: nonexistent in zstd-sys
 - pkg-config, legacy: no-op in zstd-sys
Last-Update: 2023-02-02
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: zstd-safe/Cargo.toml
===================================================================
--- zstd-safe.orig/Cargo.toml
+++ zstd-safe/Cargo.toml
@@ -47,8 +47,6 @@ features = [
 
 [features]
 arrays = []
-bindgen = ["zstd-sys/bindgen"]
-debug = ["zstd-sys/debug"]
 default = [
     "legacy",
     "arrays",
@@ -56,16 +54,10 @@ default = [
 ]
 doc-cfg = []
 experimental = ["zstd-sys/experimental"]
-fat-lto = ["zstd-sys/fat-lto"]
 legacy = ["zstd-sys/legacy"]
-no_asm = ["zstd-sys/no_asm"]
 pkg-config = ["zstd-sys/pkg-config"]
-seekable = ["zstd-sys/seekable"]
 std = ["zstd-sys/std"]
-thin = ["zstd-sys/thin"]
-thin-lto = ["zstd-sys/thin-lto"]
 zdict_builder = ["zstd-sys/zdict_builder"]
-zstdmt = ["zstd-sys/zstdmt"]
 
 [lib]
 name = "zstd_safe"
